proto/pyproject.toml

```diff
@@ -59,7 +59,7 @@ proto_builds = [
 [tool.black]
 line-length = 88
 target-version = [
-  'py38',
+  'py311',
```

Comment:
Noticed that otaclient-iot-logging-server set the target version as python3.11, thus modified.